quay-enterprise/bitbucket-app.md

```diff
@@ -0,0 +1,19 @@
+# Creating an OAuth Application in BitBucket
+
+- Log into BitBucket
+- Visit the Settings page for your organization
+- Click the "OAuth" tab under "Access Management"
+- Click the Add Consumer button:
+
+<img src="img/bb-add-consumer.png" class="image-center"/>
+
+- Enter your registry's URL as the `URL`
+- Enter `https://{REGISTRY URL HERE}/oauth2/bitbucket/callback` as the `Callback URL`.
+- Grant permissions on the repositories and webhooks:
+
+<img src="img/bb-app-permissions.png" class="image-center"/>
+
+- Save the application
+- Note down the `Client ID` and `Client Secret`.
```

Comment:
"Record the *Key* and *Secret* shown in the new entry in the list of OAuth consumers after saving the application."

(the words used here should match those in the screenshot and the final sentence. Key and Secret. Sorry to boot it back.)